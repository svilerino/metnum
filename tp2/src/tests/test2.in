0 0.85 1 tests/test2.txt 0.00001
