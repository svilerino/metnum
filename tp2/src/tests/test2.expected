0.038
0.2825
0.0656
0.056
0.2289
0.3281
